README.md

```diff
@@ -7,11 +7,11 @@ This tool is meant to help visualize potential flow theory using an intuitive in
 You can launch a web instance of the application through [this link](https://ae2130i-potential-flow-tool.streamlit.app/). If the application hasn't been used in a while, the booting process might take several minutes.
 
 The functionality is very straightforward:
-* Use the `Clear Flow` and `Update Flow` buttons to remove or update all flow elements.
+* Use the `Clear Flow` and `Draw Flow` buttons to clear or draw the flow elements. Make sure to draw the plot whenever you add new flow elements.
 * Use the `Graphing` tab to change the domain and mesh of the plots, and to adjust the looks of the plots.
```

Comment:
I would put the Graphing tab last, to avoid overcomplicating from the start.
Maybe we should rename it to `Options` or `Settings`, so that it is immediately clear

Reply:
Sounds good!